tests/multio/action/statistics/test_inverse_difference.cc

```diff
@@ -42,28 +40,28 @@ const std::string actionsInverseDifference(
                     "use-current-time": "false",
                     "step-frequency": 1,
                     "time-step": 3600,
-                    "initial-condition-present": "true"
+                    "initial-condition-present": "true",
+                    "solver-reset-accumulate-fields-every": "never"
```

Comment:
this "solver-reset-accumulate-fields-every" is important ... but should be default to never for difference